pkg/pillar/cmd/usbmanager/usbmanager.go

```diff
@@ -81,7 +82,7 @@ func Run(ps *pubsub.PubSub, loggerArg *logrus.Logger, logArg *base.LogObject, ar
 	}
 	log.Functionf("processed Vault Status")
 
-	currentHypervisor := hypervisor.CurrentHypervisor()
+	currentHypervisor := domainmgr.CurrentHypervisor()
```

Comment:
It probably makes sense to have this pick up and use the boot time hypervisor by parsing /run/eve-hv-type.

Reply:
But let's do that in a separate PR.